src/utils.c

```diff
@@ -129,6 +129,7 @@ void avifArrayPop(void * arrayStruct)
 {
     avifArrayInternal * arr = (avifArrayInternal *)arrayStruct;
     --arr->count;
```

Comment:
should we bail if count == 0 when this function is called?

if we are sure that could will always be > 0, then maybe add an assert?

Reply:
It is a programming error to call `avifArrayPop()` on an empty array. So I don't want to silently bail out if arr->count == 0. We can call `abort()` or print an error message.